src/libraries/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -15,6 +15,8 @@ namespace System.Runtime.Serialization
     using System.Linq;
     using Xml.Schema;
     using System.Collections.Concurrent;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Diagnostics;
 
 #if USE_REFEMIT
     public abstract class DataContract
```

Comment:
Is this class really public for some platform? I looked https://docs.microsoft.com/en-us/dotnet/api/?view=net-5.0&term=DataContract and didn't show up.

Reply:
These defines should be removed. See https://github.com/dotnet/runtime/issues/41512. It isn't public for anything we care about. I'm totally guessing, but I think these needed to be marked "public" in the implementation for certain platforms, but I don't see they were ever public in the ref.